apiserver/common/machine.go

```diff
@@ -103,6 +103,21 @@ func destroyMachines(st stateInterface, force bool, maxWait time.Duration, ids .
 	return apiservererrors.DestroyErr("machines", ids, errs)
 }
 
+// ModelApplicationInfo returns information about applications.
+func ModelApplicationInfo(st ModelManagerBackend) (applicationInfo []params.ModelApplicationInfo, _ error) {
```

Comment:
It's weird that this lives in `machine.go`

Reply:
agreed